redhat-operators/codeready-workspaces/codeready-workspaces.1.1.0.csv.yaml

```diff
@@ -0,0 +1,370 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: crwoperator.v1.1.0
+  namespace: placeholder
+  annotations:
```

Comment:
If this operator has a public repository, please add a link to it under the `metadata.annotations.repository` field.

Reply:
Fixed